### DISCUSSION TITLE
Could I use this library in the production environment for my project?

### DISCUSSION THREAD

**Discussion by cw1997** [Aliheym/typeorm-transactional#36] [Q&A]
From the version code I have seen, the major version is ZERO, so could I use this library in the production environment for my project?

**Comment by adrianvdh** [comment]
@Aliheym Please respond as I intend to do the same

**Reply by Aliheym** [reply]
Hi, guys. 

I've successfully used this library for two of my real projects, I would advise against using it in a production environment for others at this time. 

In the upcoming one to two months, I plan to release a stable major version. This release might introduce modifications to the public API and will also transition to using Async Local Storage by default.

**Reply by ofuochi** [reply]
@Aliheym is it production ready now?

**Reply by maks-rafalko** [reply]
@Aliheym also interested, is it production ready now?
